template/golang-http/main.go

```diff
@@ -60,11 +61,21 @@ func makeRequestHandler() func(http.ResponseWriter, *http.Request) {
 func main() {
 	s := &http.Server{
 		Addr:           fmt.Sprintf(":%d", 8081),
-		ReadTimeout:    3 * time.Second,
-		WriteTimeout:   3 * time.Second,
+		ReadTimeout:    getDuration("write_timout", 10*time.Second),
+		WriteTimeout:   getDuration("read_timout", 10*time.Second),
 		MaxHeaderBytes: 1 << 20, // Max header of 1MB
 	}
 
 	http.HandleFunc("/", makeRequestHandler())
 	log.Fatal(s.ListenAndServe())
 }
+
+// getDuration returns a duration value from the environment or returns the default
```

Comment:
Looks fine, but where was this copied from?

What about the armhf variants and the golang-middleware template too?

Please see issue #9

Reply:
Inspired by this:
https://github.com/openfaas-incubator/of-watchdog/blob/ba8afec031da6315e020026c809e48e45c9ba248/config/config.go#L107-L116

But simplified since the complexity of a map or config object isn't justified here.

Reply:
I think the map is fine, it allows testability without messing with `os.Setenv()`.. but this is also OK. Just wanted to see it documented where the code was copied from as a comment.

Also related: https://github.com/openfaas/faas/blob/d8d6349b2b86d6f913a73cacb91ffc6017b2def6/watchdog/readconfig.go#L31